n3fit/src/n3fit/model_gen.py

```diff
@@ -669,19 +669,18 @@ def pdfNN_layer_generator(
     else:
         sumrule_layer = lambda x: x
 
+    compute_preprocessing_factor = Preprocessing(
+        flav_info=flav_info,
+        input_shape=(1,),
+        name="preprocessing_factor",
+        seed=42,  # TODO Aron: figure out what to do here
```

Comment:
Here it is absolutely impossible to keep the same random result than before, so just do `seed=replica_seed + number_of_layers` or whatever takes you out from a seed previously used in a replicable manner and that's it.